inception-ui-search/src/main/java/de/tudarmstadt/ukp/inception/app/ui/search/sidebar/SearchAnnotationSidebar.java

```diff
@@ -18,7 +18,7 @@
 package de.tudarmstadt.ukp.inception.app.ui.search.sidebar;
 
 import static de.tudarmstadt.ukp.clarin.webanno.api.annotation.util.WebAnnoCasUtil.getAddr;
-import static de.tudarmstadt.ukp.clarin.webanno.api.annotation.util.WebAnnoCasUtil.selectSingleFsAt;
+import static org.apache.uima.fit.util.CasUtil.selectSingleAt;
```

Comment:
Not sure why this is necessary? This changes the behavior of the program and has nothing to do with the overall intention of the pull request (explicitly declaring variables).

Reply:
This change has been remover